### PR TITLE
:memo: README.md

### PR DESCRIPTION
**Pull request by fabienjuif**
https://github.com/Dedetat/powerpuff-yourself/tree/README

### CODE REVIEW

**Comment by fabienjuif** [comment]
Hmm, styles are not all there :(

**Comment by fabienjuif** [comment]
What I wanted is 
image

File: README.md [hunk 1]
```diff
@@ -1 +1,66 @@
-# mst-conf
+# Pimp my Powerpuff
+Welcome to our source speech repository about [mobx-state-tree](https://github.com/mobxjs/mobx-state-tree) and powerpuffs.
+
+ - All informations are in the README 👌
+ - Source code of our livecoding are described here too ! 📖
```
Comment (delphinemillet) [inline README.md]: Source code of our livecoding `is`

File: README.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+ - [Devfest Nantes, France - 2017](https://devfest2017.gdgnantes.com/)
+
+## Livecoding
+### Model
+> This is about creating our first model
+
+#### [#12](https://github.com/Dedetat/powerpuff-yourself/pull/12/files) - Create the first model and instanciate it
+> Write our first model about a **Powerpuff**, we can call it `Powerpuff` (damn, clever)
+ 1. We import `types` from `mobx-state-tree`, this is the most common used API
+ 2. `types` is used to ... types fields 👌, we can already use it to describe models ! And  `types.model` is the way to go !
```
Comment (delphinemillet) [inline README.md]: to ... `type` fields

File: README.md [hunk 3]
```diff
@@ -1 +1,66 @@
...
+ 3. Here we create a model with only one field, a `name` which is a string !
+ 4. Ok, let's instanciate our model with `Powerpuff.create({})`
+    - `Powerpuff` is our model
+    - `create` means we instanciate a new tree, a new instance
+    - `{}` is the snapshot, the raw data, to use to instanciate a new powerpuff !
+ 5. Let's print it with a `console.log` !
+    - You can see we use `powerpuff.toJSON()` to convert our tree to a snapshot.
+ 6. 🤔 oups, it doesn't work at all ! This is because, by default, a type is required.
+ 7. Let's instanciate our powerpuff, it will be... hm... `Rebelle` (this is Blossum in french, sorry)
```
Comment (delphinemillet) [inline README.md]: Blossom

File: README.md [hunk 4]
```diff
@@ -1 +1,66 @@
...
+ 8. Print it
+ 9. it works :tada:
```
Comment (delphinemillet) [inline README.md]: majuscule

File: README.md [hunk 5]
```diff
@@ -1 +1,66 @@
...
+ 1. We import `types` from `mobx-state-tree`, this is the most common used API
+ 2. `types` is used to ... types fields 👌, we can already use it to describe models ! And  `types.model` is the way to go !
+ 3. Here we create a model with only one field, a `name` which is a string !
+ 4. Ok, let's instanciate our model with `Powerpuff.create({})`
+    - `Powerpuff` is our model
+    - `create` means we instanciate a new tree, a new instance
+    - `{}` is the snapshot, the raw data, to use to instanciate a new powerpuff !
+ 5. Let's print it with a `console.log` !
+    - You can see we use `powerpuff.toJSON()` to convert our tree to a snapshot.
+ 6. 🤔 oups, it doesn't work at all ! This is because, by default, a type is required.
```
Comment (delphinemillet) [inline README.md]: a `field` is required

**Review by delphinemillet — CHANGES_REQUESTED** [review]
revision

**Comment by delphinemillet** [comment]
it seems that Github sanitize markdown before convert it to Markdown so style attributes are ignored

**Comment by fabienjuif** [comment]
I can remove our avatar then. 
What do you think ?

**Comment by delphinemillet** [comment]
it's ok to me 👍

File: README.md [hunk 6]
```diff
@@ -1 +1,50 @@
...
+
+## Events
+ - [Devfest Nantes, France - 2017](https://devfest2017.gdgnantes.com/)
+
+## Livecoding
+### Model
+> This is about creating our first model
+
+#### [#12](https://github.com/Dedetat/powerpuff-yourself/pull/12/files) - Create the first model and instanciate it
+> Write our first model about a **Powerpuff**, we can call it `Powerpuff` (damn, clever)
```
Comment (delphinemillet) [inline README.md]: model `of` Powerpuff

**Review by delphinemillet — APPROVED** [review]

**Pull request merged**